server/unittools.cpp

```diff
@@ -2586,53 +2578,41 @@ void kill_unit(struct unit *pkiller, struct unit *punit, bool vet)
           notify_player(player_by_number(i), ptile, E_UNIT_LOST_DEF,
                         ftc_server,
                         // TRANS: "Cannon ... the Polish Destroyer."
-                        _("%s lost to an attack by the %s %s."), punit_link,
-                        nation_adjective_for_player(pvictor), pkiller_link);
+                        _("%s lost when the %s %s defeated your %s."),
```

Comment:
> Warriors lost when the Celtiberian Frigate defeated our Warriors.

--> this sounds like two different warriors.

Reply:
hm... Should this notify_player exist at all? If your unit loss the battle and you only have one unit on that tile, you already received a message about this at this stage. But I didn't see this behaviour in the test-game... I'll try again.